Sources/SentryCrash/Recording/SentryCrashReport.c

```diff
@@ -1647,6 +1652,66 @@ writeDebugInfo(const SentryCrashReportWriter *const writer, const char *const ke
     writer->endContainer(writer);
 }
 
+static void
+writeScopeJson(const SentryCrashReportWriter *const writer)
+{
+    SentryCrashScope *scope = sentrycrash_scopesync_getScope();
+    writer->beginObject(writer, SentryCrashField_Scope);
+    {
+        if (scope->user) {
+            addJSONElement(writer, "user", scope->user, false);
+        }
+        if (scope->dist) {
+            addJSONElement(writer, "dist", scope->dist, false);
+        }
+        if (scope->context) {
+            addJSONElement(writer, "context", scope->context, false);
+        }
+        if (scope->environment) {
+            addJSONElement(writer, "environment", scope->environment, false);
+        }
+        if (scope->tags) {
+            addJSONElement(writer, "tags", scope->tags, false);
+        }
+        if (scope->extras) {
+            addJSONElement(writer, "extra", scope->extras, false);
+        }
+        if (scope->fingerprint) {
+            addJSONElement(writer, "fingerprint", scope->fingerprint, false);
+        }
+        if (scope->level) {
+            addJSONElement(writer, "level", scope->level, false);
+        }
+
+        if (scope->breadcrumbs) {
+
+            bool areThereBreadcrumbs = false;
+            for (int i = 0; i < scope->maxCrumbs; i++) {
+                if (scope->breadcrumbs[i]) {
+                    areThereBreadcrumbs = true;
+                    break;
+                }
+            }
+
+            if (areThereBreadcrumbs) {
+                writer->beginArray(writer, "breadcrumbs");
```

Comment:
probably not necessary to skip this, but its not wrong either.

Reply:
I rather keep it not to have an empty breadcrumbs array in the JSON.